lms/templates/navigation.html

```diff
@@ -79,9 +79,7 @@ <h1 class="logo">
         <a href="#" class="dropdown" aria-haspopup="true" aria-expanded="false"><span class="sr">${_("More options dropdown")}</span><i class="fa fa-sort-desc" aria-hidden="true"></i></a>
         <ul class="dropdown-menu" aria-label="More Options" role="menu">
           <%block name="navigation_dropdown_menu_links" >
-            % if settings.MKTG_URL_LINK_MAP.get('FAQ'):
```

Comment:
Same comment as before: I don't think this should be removed. The "Help" link is shown on edx.org and it is only on devstack that it seems to be a no-op.